quickstarts/File_API.ipynb

```diff
@@ -44,7 +44,7 @@
     },
```

Comment:
<p>Nit: link to your upload files notebook, instead of just the examples folder.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done